sdk/python/kfp/containers/_build_image_api.py

```diff
@@ -80,6 +80,12 @@ def build_image_from_working_dir(image_name: str = None, working_dir: str = None
         timeout: Optional. The image building timeout in seconds.
         base_image: Optional. The container image to use as the base for the new image. If not set, the Google Deep Learning Tensorflow CPU image will be used.
         builder: Optional. An instance of ContainerBuilder or compatible class that will be used to build the image.
+          The default builder uses "kubeflow-pipelines-container-builder" service account in "kubeflow" namespace. It works with Kubeflow Pipelines clusters installed using Google Cloud Marketplace or Standalone with version > 0.4.0.
```

Comment:
> in "kubeflow" namespace
in namespace installed Kubeflow Pipelines?

Reply:
that's right! thanks for pointing out

Reply:
Fixed